checks/cs50/2017/x/scratch/checks.py

```diff
@@ -0,0 +1,133 @@
+import json
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import *
+
+class Scratch(Checks):
```

Comment:
@crossroads1112, we recently discussed adding a `@checks` decorator so that we needn't just assume the first (or only) class in a file is the one we want for checks. But now that we're adding `__init__.py`, is that moot, since we can rely on `__init__.py` to export the class we care about?